modules/permutiveRtdProvider.js

```diff
@@ -283,7 +272,12 @@ function getDefaultBidderFn (bidder) {
     }
   }
 
-  return bidderMap[bidder]
+  // On no default bidder just return the same bid as passed in
+  function bidIdentity(bid) {
+    return bid
+  }
+
+  return bidderMap[bidder] || bidIdentity
```

Comment:
So we don't break the public API of our module. `rubicon` did return a default bidder but since it was removed it would now return undefined.  On [line 208](https://github.com/prebid/Prebid.js/pull/9555/files#diff-9476fbe01d356d6cb1e8a1a00bcdda0576f4b82641ad355f61f3252ddd821492R208), we would then pass the default handler to any publisher defined _custom_ handler. Ideally, they would check there is a default handler that exists first (i.e `if (defaultFn) { defaultFn() }`) but we can't guarantee any custom script on a publisher's page actually does this check. Hence if the invoke an undefined, it will throw an error. So we now provide a fallback which does nothing to the bid.